docs/design/faireness-effectiveness-statistic.md

```diff
@@ -0,0 +1,88 @@
+# Volcano Statistic And Metrics
+
+@jamesjiang; April 24, 2020
+
+## Motivation
+
+Currently, Volcano does not have statistics for cluster admin to evaluate the job wait fairness and every user and job receive the same treatment.
+
+Volcano also dose not have statistics for platform administrator to evaluate resources usage when job wait latency is so high. 
+
+Cluster admin can get the result when he finish tuning plugins of cluster to fit his own 
+
+
+## Design
+
+1. Fairness is very important for cluster admin to keep a balance for preempt and resource reserve.
+2. Effectiveness is also an important factor for cluster admin to consider for resource reserve.
+
+### Fairness
+
+> When scheduling is in high latency and retry rates and e2e_scheduling_latency_standard_deviation is large 
+
+1. job_estimate_avg_error_percentage:  Job Running and estimate difference value ` avg(|job_running_duration - job_estimate| / job_running_duration)`
+2. e2e_scheduling_latency_coefficient_of_variation: Job Lagency Coefficient Of Variation
+   `e2e_scheduling_latency_coefficient_of_variation = stddev_over_time(e2e_scheduling_latency)/avg(e2e_scheduling_latency)`
+3. job_retry_rates: job_retry_count / job_schedule_count
```

Comment:
How about add a dimension to reflect the distribution of delay

Reply:
OK, the origin latency is about the schedule level not the job level, i will add a job level later.